src/js/video.js

```diff
@@ -308,6 +312,8 @@ class Video {
 		}
 
 		this.videoEl.src = this.rendition && this.rendition.url;
+		this.guidance.removeBanner();
+		this.videoEl.addEventListener('playing', this.showGuidanceBanner.bind(this), { once: true });
```

Comment:
`addEventListener`'s once option is not supported in IE11 and not supported by polyfill.io either. You will likely need another solution for IE11 to use.